kubeshop/testkube-dashboard
# feat: use react-ga4 instead of ga-4-react library

## Changes

- use `react-ga4` library instead of `ga-4-react`
- send common `gtm.js` event when initialized
- send all GTM consents when initializing
- allow multiple GA/GTM tracking IDs (comma separated in environment variable)

## Fixes

-

## How to test it

- https://testkube-dashboard-git-dawid-fixuse-diffe-6d3a0a-testkube-cloud.vercel.app/?~google_analytics_id=GTM-PQK4DKN,G-G7HWN1EDK5&~disable_telemetry=false
- connect to cluster with enabled telemetry
- accept cookies
- see `dataLayer` in the console
- see `$$('script').map(s => s.src).filter(x => /(GTM|G)-/.test(x))`

## screenshots

-

## Checklist
